### DISCUSSION TITLE
Training - Missing format choice input

### DISCUSSION THREAD

**Discussion by Dave86ch** [oobabooga/text-generation-webui#3478] [Q&A]
Hello,

I'm attempting to train a model for the first time and need some guidance. 
I'm using this dataset format:

```json
{
  "id": "0",
  "text": "From my grandfather Verus I learned good morals and the government of my temper.",
  "source": "meditations"
}
{
  "id": "1",
  "text": "From the reputation and remembrance of my father, modesty and a manly character.",
  "source": "meditations"
} 
```

However, I'm receiving an error in the web UI: 
'Missing format choice input, cannot continue.' 
Could you assist?

**Comment by Monty1122** [comment]
this error is to do with the Data Format choice in the Training Tab, Formatted Dataset section...you need to make a choice in the dropdown in the UI...however im having the same problem trying to train Falcon 7B as its not clear which of the dropdown choices to pick for Falcon ...would be great to hear from anybody who has used Falcon to get their view on what choice to select.
